labonneboite/conf/common/overrides/development.py

```diff
@@ -1,7 +1,7 @@
 import flask.logging
 
 DEBUG = True
-SERVER_NAME = None
+SERVER_NAME = 'localhost:5000'
```

Comment:
Hmmm ce changement ne me plaît pas beaucoup. Il signifie qu'on ne pourra plus accéder au serveur de dev via `127.0.0.1`, ni `0.0.0.0`, ni via un autre port. Je viens de tester et j'ai une erreur "La page que vous avez demandée n'existe pas."

Du coup pour moi les seules solutions sont:
1. Utiliser le path de la requête courante et vérifier que l'hôte est identique à celui de next_url. C'est ce que j'avais suggéré sur slack et c'est mon approche préférée puisqu'elle ne nécessite pas de définir de nouveau setting ni d'interpréter différemment un setting existant.
2. Ne pas faire de vérification de l'hôte lorsque `SERVER_NAME is None`, mais ça veut dire alors que la fonctionnalité que tu développes n'est pas testable en dev.

Evidemment, je préfère la 1...

Reply:
will do a mix of 1 and 2

on utilise `request.full_path` qui évite la question du domaine